www/docs/en/dev/guide/platforms/ios/index.md

```diff
@@ -72,192 +60,124 @@ There are two ways to download Xcode:
   which requires registration as an Apple Developer.
 
 Once Xcode is installed, several command-line tools need to be enabled
-for Cordova to run. From the __Xcode__ menu, select __Preferences__,
-then the __Downloads__ tab. From the __Components__ panel, press the
-__Install__ button next to the __Command Line Tools__ listing.
+for Cordova to run. From the command line, run: 
+``` xcode-select --install``` 
 
-## Install Deploy Tools
+### Deployment Tools
 
-Run from comman-line terminal:
+The [ios-sim](https://www.npmjs.org/package/ios-sim) and 
+[ios-deploy](https://www.npmjs.org/package/ios-deploy) tools - allows you
+to launch iOS apps into the iOS Simulator and iOS Device from the command-line.
+
+To install them, run the following from command-line terminal:
 
         $ npm install -g ios-sim
         $ npm install -g ios-deploy
 
-## Create a New Project
-
-Use the `cordova` utility to set up a new project, as described in The
-Cordova [The Command-Line Interface](../../cli/index.html). For example, in a source-code directory:
+## Project Configuration
 
-        $ cordova create hello com.example.hello "HelloWorld"
-        $ cd hello
-        $ cordova platform add ios
-        $ cordova prepare              # or "cordova build"
+Installing Xcode will mostly set everything needed to get started.
```

Comment:
We can also give a link here to the CLI, something like 'You should be able to create a cordova project now. For more details refer to the CLI"